src/metrics/nodemetadata/metadata_req.go

```diff
@@ -0,0 +1,50 @@
+package nodemetadata
```

Comment:
what do you think about moving this outside of `metrics`? imho we should keep in metrics just the code that is not related to rumor. So that if in the future we use libp2p directly we can reuse everything as it is without any changes.

Reply:
Yep, was a temporary place, moving it to the actor/host package